vendor/engines/secure_rooms/spec/controllers/secure_rooms_api/scans_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe SecureRoomsApi::ScansController do
+  context "scan" do
+    before do
+      name = Settings.secure_rooms_api.basic_auth_name
+      password = Settings.secure_rooms_api.basic_auth_password
+      encoded_auth_credentials = ActionController::HttpAuthentication::Basic.encode_credentials(name, password)
+      request.env['HTTP_AUTHORIZATION'] = encoded_auth_credentials
+      post :scan
+    end
+
+    it "denies entry until we add logic" do
+      expect(response.code).to eq "403"
```

Comment:
Looks like the issue is more around the credentials than the behavior itself. Given that, part of me likes the readability of `to be_forbidden`, but the error is much easier to troubleshoot this way.

Upon googling this just now, I discovered rspec-rails has a `have_http_status` matcher that should allow us to do `expect(response).to have_http_status(:forbidden)`.

https://www.relishapp.com/rspec/rspec-rails/docs/matchers/have-http-status-matcher

I think that gives us the best of both worlds: "expected the response to have status code :forbidden (403) but it was :unauthorized (401)"

I'm going to use this more often.